Document shouldFinalize for sketches that have the parameter (#13524)
apache/druid: docs/development/extensions-core/datasketches-hll.md

  -->

> This module provides Apache Druid aggregators for distinct counting based on HLL sketch from [Apache DataSketches](https://datasketches.apache.org/) library. At ingestion time, this aggregator creates the HLL sketch objects to be stored in Druid segments. At query time, sketches are read and merged together. In the end, by default, you receive the estimate of the number of distinct values presented to the sketch. Also, you can use post aggregators to produce a union of sketch columns in the same row.

Reviewer: ```suggestion
This module provides Apache Druid aggregators for distinct counting based on HLL sketch from [Apache DataSketches](https://datasketches.apache.org/) library. At ingestion time, this aggregator creates the HLL sketch objects to store in Druid segments. By default, Druid reads and merges sketches at query time. The default result is 
 the estimate of the number of distinct values presented to the sketch. You can also use post aggregators to produce a union of sketch columns in the same row.
```

> You can use the HLL sketch aggregator on columns of any identifiers. It will return the estimated cardinality of the column.

Reviewer 2: Not sure what `columns of any idenfiers` means. I guess a rephrase might help
```suggestion
You can use the HLL sketch aggregator on any column to estimate its cardinality.
```

To use this aggregator, make sure you [include](../../development/extensions.md#loading-extensions) the extension in your config file:

```
druid.extensions.loadList=["druid-datasketches"]
```

For additional sketch types supported in Druid, see [DataSketches extension](datasketches-extension.md).

## Aggregators

|Property|Description|Required?|
|--------|-----------|---------|

> |`type`|This string should be [`HLLSketchBuild`](#hllsketchbuild-aggregator) or [`HLLSketchMerge`](#hllsketchmerge-aggregator).|yes|

Reviewer: ```suggestion
|`type`|Either [`HLLSketchBuild`](#hllsketchbuild-aggregator) or [`HLLSketchMerge`](#hllsketchmerge-aggregator).|yes|
```

> |`name`|A string for the output (result) name of the calculation.|yes|

Reviewer: ```suggestion
`name`|String representing the output column to store sketch values.|yes|
```

> |`fieldName`|A string for the name of the input field.|yes|

Reviewer: ```suggestion
|`fieldName`|The name of the input field.|yes|
```
